integration/test-network/README.md

```diff
@@ -0,0 +1,48 @@
+# Setup the FPC test network
+
+The FPC test network builds on the test-network provided by [fabric-samples](https://github.com/hyperledger/fabric-samples).
+
+We provide fabric-samples as a submodule in `$FPC_PATH/integration/test-network/fabric-samples`.
+
+Make sure you have installed [yq](https://github.com/mikefarah/yq).
+
+Before you start the network make sure you build ercc and ecc_mock. In
+
+```bash
+cd $FPC_PATH/ercc
+make && make docker
+
+cd $FPC_PATH/ecc_mock
+make && make docker
+```
+
+## Prepare network
+
+Go to `$FPC_PATH/integration/fabric-samples` and follow the [instructions](https://hyperledger-fabric.readthedocs.io/en/latest/install.html) to get the fabric binaries and docker images.
+
+```bash
+cd $FPC_PATH/integration/test-network/fabric-samples
+curl -sSL https://bit.ly/2ysbOFE | bash -s -- -s
+```
+ 
+```bash
+cd $FPC_PATH/integration/test-network
+./setup.sh
+```
+
+Start network:
+```bash
+cd $FPC_PATH/integration/test-network/fabric-samples/test-network
+./network.sh up createChannel -c mychannel -ca
+```
+
+Install FPC components:
+```bash
+cd $FPC_PATH/integration/test-network
+./installFPC.sh
+```
+
+Start FPC container
+```bash
+docker-compose up
```

Comment:
this doesn't work but that's because mock_ecc doesn't build (yet) which i understand is because parts of #450 and/or #452 are missing. So we can resolve that as part of #452 ....